pypdf/_page_labels.py

```diff
@@ -173,3 +183,78 @@ def index2label(reader: PdfReaderProtocol, index: int) -> str:
         )
     # TODO: Implement /Kids and /Limits for number tree
     return str(index + 1)  # Fallback
+
+
+def nums_insert(
```

Comment:
The current structure of the PR and, in particular, the placement of the `nums` functions in `_page_labels.py`